docs/ensembl-help/using-ensembl/ensembl-apps/genome-browser/navigation.md

```diff
@@ -13,12 +13,32 @@ status: draft
 
 # Genome browser navigation
 
-You can navigate the Genome browser using mouse actions and the Navigation bar.
+You can navigate the Genome browser using a combination of the Navigation bar and mouse or track pad actions.
 
-To navigate with your mouse, move it into the browser image. You can move left or right by clicking and dragging the mouse. You can zoom in or out using the scroll wheel or two-finger scrolling on a trackpad – you will zoom towards the position of your cursor. The blue coordinates on the right of the Navigation bar indicate your current genomic position.
+## How to adjust position using the Navigation bar
 
-You can expand the Navigation bar by clicking on the blue coordinates. This will now display the chromosome with our region highlighted and some navigation icons. Zoom in and out using the plus and minus icons. Move left and right using the arrows. Click on change on the right to enter new genomic coordinates. Hide the navigation by clicking on the blue coordinates again.
+The blue coordinates on the right of the Navigation bar, next to the chromosome number, indicate your current genomic position in bp.
 
 ![The Navigation bar](navigation_bar.png)
 
-The coordinates of the Focus entity are shown on the left of the Navigation bar. As you move around using either the Navigation bar or the mouse actions, the Focus entity coordinates remain constant. To return to the region of your Focus entity, click on the target icon on the left, which will be blue when you are away from the Focus entity. When you are looking at the Focus entity, the target icon will be grey.
+You can expand the Navigation bar by selecting on the blue coordinates to reveal a set of navigation controls. 
+
+Select 'Change' on the right to enter new genomic coordinates.
+
+Use the plus and minus icons to zoom in and out and the arrows to move left and right.
+
+Hide the navigation by selecting the blue coordinates to the right of the Chromosome number again.
+
+
+## How to adjust position using a mouse or track pad
+
+To navigate using a mouse, move the cursor into the browser. To move left or right by click and drag the mouse in either direction. 
+
+You can zoom in or out using the scroll wheel or two-finger scrolling on a trackpad – you will zoom towards the position of your cursor.
+
+
+## Return to the original position Focus entity
+
+To return to the region of your Focus entity, select the target icon on the far left of the Navigation bar, which will be blue when you are away from the Focus entity. When you are looking at the Focus entity, the target icon icon will be grey.
```

Comment:
The target icon doesn't at the moment, turn grey. Delete this sentence until it does ?